src/tests/ftest/util/soak_utils.py

```diff
@@ -634,9 +634,12 @@ def launch_reboot(self, pools, name, results, args):
     # If all ranks "joined", issue reintegrate for all pool on all ranks and wait for
     #    rebuild to complete
     # Update multiprocessing queue with results and args
+    # pylint: disable=too-many-nested-blocks,too-many-branches
```

Comment:
To avoid this, it would be better if the `if name == something: ... elif name == something_else:` code called sub functions for each harasser.

Reply:
+1 but this is a backport so we should do that in master first